index.html

```diff
@@ -0,0 +1,458 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header class="header">
+      <h1 id="logo">Aziz Mejri</h1>
+
+      <nav class="nav">
+        <ul class="nav-menu desktop">
+          <li class="nav-item">
+            <a href="#home">Portfolio</a>
+          </li>
+          <li class="nav-item">
+            <a href="#about">About</a>
+          </li>
+          <li class="nav-item">
+            <a href="#contact">Contact</a>
+          </li>
+          <li class="nav-item desktop">
+            <a href="#mail">
+              <img src="./assets/images/icons/Icon-Mail.svg" alt="mail" />
+            </a>
+          </li>
+        </ul>
```

Comment:
On the client side, I have noticed the nav links are not given a colour according to the [requirements.](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.)

I suggest you add the required colour as this will improve the look of the UI and better the chances of a potential recruiter liking your app.